### PR TITLE
added new location: Barcelona

### PR DESCRIPTION
**Pull request by lpmayos**

### CODE REVIEW

File: www/config.yml [hunk 1]
```diff
@@ -933,6 +933,18 @@ chapters:
   twitter: Pyladiesowerri
   website: IN
   whatsapp: bit.ly/pyladiesimo
+- email: pyladies-bcn@googlegroups.com
```
Comment (lorenanicole) [inline www/config.yml]: We require that all official `PyLadies` business be done via the official `@pyladies.com` email address.

Comment (lpmayos) [reply www/config.yml]: And as a community, how do we get an oficial barcelona@pyladies.com email address?

Comment (lorenanicole) [reply www/config.yml]: Why, thought you'd never ask! There is this email actually already `bcn@pyladies.com`! Can you ping me in the PyLadies Slack to get this password reset - https://slackin.pyladies.com/?

Comment (lorenanicole) [reply www/config.yml]: You can then register your chapter in the PyLadies chapter directory! See: https://www.pyladies.com/blog/Important-PyLadies-Update/important-pyladies-update/ about registration.

Comment (lpmayos) [reply www/config.yml]: Done! :D

**Pull request merged**